### ISSUE TITLE
Install button in Installation process remains unresponsive using Wine

### ISSUE THREAD

**Issue by sibwy** [webyog/sqlyog-community#2123]
With Wine 1.8 and 1.9.22, the ‘Install’ button in the installation process is greyed out. The button remains greyed out even on selecting an installation path using the ‘Browse’ button.

The user report can be checked in HS >> 231679

**Comment by peterwy** [comment]
Is it possible to install from command line?

**Comment by peterwy** [comment]
After internal discussion it seems that the folder /home/username/.wine/drive_c/Program Files/SQLyog/ is created, but it is emply when installing from a console

But it probably does not matter as 
1) we searched online and found that it could be due to the wine not upgraded correctly.
2) So we uninstalled wine completely using sudo apt remove —purge wine*
3) and then installed wine again. The ‘Install’ button works fine now.

So it simply seems to be a Wine corruption in Ubuntu. And this corruption seems not uncommon as both user and we experienced this. It could be caused by a bad "apt" package installed earlier by an automated update routine..
